src/game-status/game.status.js

```diff
@@ -0,0 +1,36 @@
+const { MAX_CHANCES, GAME_STATUS } = require("../consts");
+const { isArrayOfStrings, isNumber } = require("../utils/validation.utils");
+const { loseStatus } = require("./lose.status");
+const { winStatus } = require("./win.status");
+
+/**
+ * a function that gets chosen user color baord, guess color board, and number of options used.
+ * returns game status and data if game is lost, won or resumes
+ * @param {number} optionsUsed
+ * @param {[string]} chosenUserBoard
+ * @param {[string]} computerBoard
+ * @returns {{ status:number, data: { hit:number, bullseye:number } }| null}
+ */
+
+module.exports.getGameStatus = (
+  optionsUsed,
+  chosenUserBoard,
+  computerBoard
+) => {
+  if (
+    !isArrayOfStrings(chosenUserBoard) ||
+    !isArrayOfStrings(com) ||
```

Comment:
1. why not create function that check what should be the user option? 
like: `isValidOptionPattern` and call it for chosenUserBoard and also for com
2. what is com? where is from? not a good name

Reply:
what do you mean to check option patterns? you mean the strings inside the array strings?